trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/VLM.py

```diff
@@ -61,55 +80,71 @@ def VLM(conditions,settings,geometry):
         fineness.tail                          [Unitless]
         
-       settings.number_spanwise_vortices       [Unitless]  <---|
-       settings.number_chordwise_vortices      [Unitless]  <---|
+    settings.number_spanwise_vortices          [Unitless]  <---|
+    settings.number_chordwise_vortices         [Unitless]  <---|
                                                                |--Either/or; see generate_vortex_distribution() for more details
-       settings.wing_spanwise_vortices         [Unitless]  <---|
-       settings.wing_chordwise_vortices        [Unitless]  <---|
-       settings.fuselage_spanwise_vortices     [Unitless]  <---|
-       settings.fuselage_chordwise_vortices    [Unitless]  <---|  
+    settings.wing_spanwise_vortices            [Unitless]  <---|
+    settings.wing_chordwise_vortices           [Unitless]  <---|
+    settings.fuselage_spanwise_vortices        [Unitless]  <---|
+    settings.fuselage_chordwise_vortices       [Unitless]  <---|  
+       
+    settings.use_surrogate                     [Unitless]
+    settings.propeller_wake_model              [Unitless]
+    settings.discretize_control_surfaces       [Boolean] -- set to True to generate control surface panels
+    settings.use_VORLAX_matrix_calculation     [boolean]
+    settings.floating_point_precision          [np.dtype]
        
-       settings.use_surrogate                  [Unitless]
-       settings.propeller_wake_model           [Unitless]
-       settings.discretize_control_surfaces    [Boolean] -- set to True to generate control surface panels
-       conditions.aerodynamics.angle_of_attack [radians]
-       conditions.freestream.mach_number       [Unitless]
+    conditions.aerodynamics.angle_of_attack    [radians]
+    conditions.aerodynamics.side_slip_angle    [radians]
+    conditions.freestream.mach_number          [Unitless]
+    conditions.freestream.velocity             [m/s]
+    conditions.stability.dynamic.pitch_rate    [radians/s]
+    conditions.stability.dynamic.roll_rate     [radians/s]
+    conditions.stability.dynamic.yaw_rate      [radians/s]
        
-    Outputs:                                   
-    CL                                         [Unitless]
-    Cl                                         [Unitless]
-    CDi                                        [Unitless]
-    Cdi                                        [Unitless]
-    CM                                         [Unitless]
-    CP                                         [Unitless]
+    
+    Outputs:    
+    results.
+        CL                                     [Unitless]
+        Cl                                     [Unitless]
+        CDi                                    [Unitless]
+        Cdi                                    [Unitless]
+        CM                                     [Unitless]
+        CP                                     [Unitless]
+        CRMTOT                                 [Unitless]
```

Comment:
Does TOT indicate something different from the items that don't have it?

Reply:
yeah, TOT indicates the total for the whole aircraft, wheras the non-TOT variables are by strip. I can make a note of that in the docstring

Reply:
Ah wait, no all of them are totals. variables with TOT are just named that way to match up with VORLAX

Reply:
noted in the docstring